Selection_Sort/Selection_Sort.ts

```diff
@@ -0,0 +1,31 @@
+// function for selection sort logic
+function Selection_Sort(numbers: number[]) : number[]{
+
+    let i: number;
+    let j: number;
+    let min: number;
+    let temp: number;
+
+    for(i = 0; i < numbers.length - 1; i++){
+        min = i;
+        for(j = i + 1; j < numbers.length; j++){
+            if(numbers[j] < numbers[min]){
+                min = j;
+            }
+        }
+        if(i !== min){
+            temp = numbers[i];
+            numbers[i] = numbers[min];
+            numbers[min] = temp;
+        }
+    }
+
+    return numbers;
+}
+
+// array of numbers (INPUT)
+const numbers: number[] = [10, 9, 8, 1, 5, 2, 4, 7, 3];
```

Comment:
Take input dynamically

Reply:
@raghav-dalmia I had discussed this with @jainaman224, he asked to take input this way, could you suggest how else to take input in typescript? (I had initially thought of using alerts in a html page)